Mathlib/Tactic/Hint.lean

```diff
@@ -72,7 +72,7 @@ def suggestion (tac : TSyntax `tactic) (msgs : MessageLog := {}) : TacticM Sugge
   let msg? ← msgs.toList.findM? fun m => do pure <|
     m.severity == MessageSeverity.information && (← m.data.toString).startsWith "Try this: "
   let suggestion ← match msg? with
-  | some m => pure <| SuggestionText.string (((← m.data.toString).drop 10).takeWhile (· != '\n'))
+  | some m => pure <| SuggestionText.string ((← m.data.toString).drop 10)
```

Comment:
```suggestion
  | some m => pure <| SuggestionText.string (((← m.data.toString).drop 10).dropWhile (· != '\n'))
```

Trying out an example, I think that it was *drop*, rather than *take*!

Reply:
Do you have an example in mind where this has any effect?